### DISCUSSION TITLE
最终需要提交 说明文档/说明书 或者 展示slides 之类的东西吗？

### DISCUSSION THREAD

**Discussion by syhien** [liusy58/NJU_AdvancedProgramming_SP22#208] [Q&A]
最终需要提交 说明文档/说明书 或者 展示slides 之类的东西吗？
就是除了 代码及可执行文件 还需要提交什么嘛

**Comment by liusy58** [comment]
> 最终需要提交 说明文档/说明书 或者 展示slides 之类的东西吗？

不需要


> 就是除了 代码及可执行文件 还需要提交什么嘛

代码足够了
